core/templates/pages/preferences-page/form-fields/preferred-languages.component.ts

```diff
@@ -94,4 +102,15 @@ export class PreferredLanguagesComponent {
       this.add(event.option);
     }
   }
+
+  onSearchInputChange(): void {
+    this.searchQuery = this.formCtrl.value;
+    this.filteredChoices = this.choices.filter(choice => {
+      let lowerSearchQuery = this.searchQuery.toLowerCase();
```

Comment:
change to `const lowerSearchQuery`. Always prefer const unless you plan to reassign the variable later

Reply:
Done